core/src/components/modal/modal.tsx

```diff
@@ -786,7 +817,17 @@ export class Modal implements ComponentInterface, OverlayInterface {
         {mode === 'ios' && <div class="modal-shadow"></div>}
 
         <div role="dialog" class="modal-wrapper ion-overlay-wrapper" part="content" ref={(el) => (this.wrapperEl = el)}>
-          {showHandle && <div class="modal-handle" part="handle"></div>}
+          {showHandle && (
+            <button
+              class="modal-handle"
+              part="handle"
+              aria-label="Activate to adjust the size of the card overlaying the dialog"
```

Comment:
What does "card overlaying the dialog" mean? The modal itself is a dialog.

Reply:
Yeah this should probably read:
> Activate to adjust the size of the card overlaying the screen.

It was adapted from the maps example of:
> Card controller half screen, adjust the size of the card overlaying the map.

Thoughts?

Reply:
Yeah I like that